hack/update-snapshots.sh

```diff
@@ -24,5 +24,5 @@ charts=`printf '%s\n' ${charts[@]} | sort -u`
 for chart in $charts; do
     # unittest needs deps to work
     helm dep build $chart
-    helm unittest --update-snapshot $chart
+    helm unittest --update-snapshot $chart --helm3
```

Comment:
Are you sure this actually is the issue? I don't even see `--helm3` as an option described here: https://github.com/helm-unittest/helm-unittest?tab=readme-ov-file#flags

Reply:
don't ask me how but the unittest helm plugin on my laptop was in version 0.2.7 while latest is 0.8.2...